### PR TITLE
PHPCS: WordAds

### PR DESCRIPTION
**Pull request by kraftbj**
Follow-up to #16193

#### Changes proposed in this Pull Request:
* PHPCS all of WordAds.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
n/a

#### Testing instructions:
* n/a

#### Proposed changelog entry for your changes:
* n/a

### CODE REVIEW

**Comment by kraftbj** [comment]
r209390-wpcom (just a minor code comment in the block, the rest of WordAds isn't shared on with the WP.com codebase).

File: modules/wordads/php/class-wordads-ccpa-do-not-sell-link-widget.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
```
Comment (jeherve) [inline modules/wordads/php/class-wordads-ccpa-do-not-sell-link-widget.php:1]: I may be missing something in that big diff, but shouldn't we require that new file in the main class now?

**Comment by dbspringer** [comment]
Thanks for taking a crack at this, going to take a look and get back to you.

File: modules/wordads/class-wordads.php [hunk 2]
```diff
@@ -313,52 +344,53 @@ public function resource_hints( $hints, $relation_type ) {
...
+	public function insert_head_meta() {
 		if ( self::is_amp() ) {
 			return;
 		}
-		$themename = esc_js( get_stylesheet() );
 		$pagetype  = intval( $this->params->get_page_type_ipw() );
 		$data_tags = ( $this->params->cloudflare ) ? ' data-cfasync="false"' : '';
 		$site_id   = $this->params->blog_id;
 		$consent   = intval( isset( $_COOKIE['personalized-ads-consent'] ) );
-		echo <<<HTML
```
Comment (dbspringer) [inline modules/wordads/class-wordads.php:325]: Is there any particular reason we're replacing the heredocs? Notably the raw HTML isn't echoing the attributes now.

Screen Shot 2020-06-25 at 11 33 00 AM

File: modules/wordads/php/class-wordads-ccpa-do-not-sell-link-widget.php (hunk 1 above)
Comment (kraftbj) [reply modules/wordads/php/class-wordads-ccpa-do-not-sell-link-widget.php:1]: I meant to do that, but ended up missing it. 183aa511b

File: modules/wordads/class-wordads.php (hunk 2 above)
Comment (kraftbj) [reply modules/wordads/class-wordads.php:325]: Fixed in d6c424a.

Replaced the heredoc to make it a bit clearer that those variables are being escaped; they weren't all before.

File: modules/wordads/class-wordads.php [hunk 3]
```diff
@@ -171,7 +196,9 @@ function init() {
 
 		if ( '/ads.txt' === $_SERVER['REQUEST_URI'] ) {
 
-			if ( false === ( $ads_txt_transient = get_transient( 'jetpack_ads_txt' ) ) ) {
+			$ads_txt_transient = get_transient( 'jetpack_ads_txt' );
```
Comment (dbspringer) [inline modules/wordads/class-wordads.php:199]: I guess this change is to make it a little clearer?

Comment (kraftbj) [reply modules/wordads/class-wordads.php:199]: Yup. WP Coding Standards don't allow setting a variable within a conditional, so breaking it out to make it clearer where they var is coming from.

**Pull request merged**